scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -201,20 +201,21 @@ task SetLock {
 
   input {
     String output_directory
-    File? service_account_json
+    String? service_account_json_path
 
     # runtime
     Int? preemptible_tries
   }
 
-  String has_service_account_file = if (defined(service_account_json)) then 'true' else 'false'
+  String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
```

Comment:
why are these a string and not a boolean?

Reply:
My guess would be that testing a boolean value in bash that was set in WDL is not particularly straightforward.